Quadriga/src/main/webapp/WEB-INF/views/workbench/addpublicpage.jsp

```diff
@@ -0,0 +1,255 @@
+											console.log("ERROR: ", e);
+										}
+									});
+ 						});
+	});
+	
+	$(function() {
+		$("#submit2")
+				.click(
+						function() {
+							var data = {};
+							var titles = $("#title2").val();
+							var desc = $("#description2").val();
+							var order = $("#order2").val();
+							var mandatory = 0;
+							if(titles.length < 1){
+								$("#title_err2").html("Title Is Mandatory");
+								mandatory = 1;
+							}else{
+								$("#title_err2").html('');
+							}
+							if(desc.length < 1){
+								$("#desc_err2").html("Description Is Mandatory");
+								mandatory =1;
+							}else{
+								$("#desc_err2").html('');
+							}
+							if(order == 0){
+								$("#order_err2").html("Order Is Mandatory");
+								mandatory =1 
+							}else{
+								$("#order_err2").html('');
+							}							
+							if(mandatory){
+								return false;
+							}
+							data["title"] = $("#title2").val();
+							data["desc"] = $("#description2").val();
+							data["order"] = $("#order2").val();
+							$
+									.ajax({
+										type : "GET",
+										url : "${pageContext.servletContext.contextPath}/auth/workbench/${ppprojectid}/addpublicpage1",
+										data : {
+											data1 : JSON.stringify(data)
+										},
+										success : function(e) {
+											$("#title2").val('');
+											$("#description2").val('');
+											$("#order2").val('');
+										},
+										error : function(e) {
+											console.log("ERROR: ", e);
+										}
+									});
+ 						});
+	});
+	
+	$(function() {
+		$("#submit3")
+				.click(
+						function() {
+							var data = {};
+							var titles = $("#title3").val();
+							var desc = $("#description3").val();
+							var order = $("#order3").val();
+							var mandatory = 0;
+							if(titles.length < 1){
+								$("#title_err3").html("Title Is Mandatory");
+								mandatory = 1;
+							}else{
+								$("#title_err3").html('');
+							}
+							if(desc.length < 1){
+								$("#desc_err3").html("Description Is Mandatory");
+								mandatory =1;
+							}else{
+								$("#desc_err3").html('');
+							}
+							if(order == 0){
+								$("#order_err3").html("Order Is Mandatory");
+								mandatory =1 
+							}else{
+								$("#order_err3").html('');
+							}							
+							if(mandatory){
+								return false;
+							}
+							data["title"] = $("#title3").val();
+							data["desc"] = $("#description3").val();
+							data["order"] = $("#order3").val();
+							$
+									.ajax({
+										type : "GET",
+										url : "${pageContext.servletContext.contextPath}/auth/workbench/${ppprojectid}/addpublicpage1",
+										data : {
+											data1 : JSON.stringify(data)
+										},
+										success : function(e) {
+											$("#title3").val('');
+											$("#description3").val('');
+											$("#order3").val('');
+										},
+										error : function(e) {
+											console.log("ERROR: ", e);
+										}
+									});
+ 						});
+	});

```

Comment:
Do you have the same function three times here? No code duplications! Have it once, but bind it several times.